frontend/src/queries/nodes/DataVisualization/Components/Variables/Variables.tsx

```diff
@@ -324,7 +344,7 @@ const VariableComponent = ({
                         onClick={() => setPopoverOpen(!isPopoverOpen)}
                         disabledReason={variableOverridesAreSet && 'Discard dashboard variables to change'}
                     >
-                        {variable.value?.toString() ?? variable.default_value?.toString()}
+                        {variable.isNull ? 'Null' : variable.value?.toString() ?? variable.default_value?.toString()}
```

Comment:
Similarily here, should the label be something like `"No value"`? Or do we think `Null` is enough? I imagine an empty string would be sufficient in everything other than String variables too